src/claims/claims.controller.ts

```diff
@@ -0,0 +1,74 @@
+import { Body, Controller, HttpStatus, Post } from '@nestjs/common';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+
+import { ClaimsService } from '~/claims/claims.service';
+import { ModifyClaimsDto } from '~/claims/dto/modify-claims.dto';
+import { ApiTransactionFailedResponse, ApiTransactionResponse } from '~/common/decorators/swagger';
+import { TransactionQueueModel } from '~/common/models/transaction-queue.model';
+import { handleServiceResult, TransactionResponseModel } from '~/common/utils';
+import { PolymeshLogger } from '~/logger/polymesh-logger.service';
+
+@ApiTags('claims')
+@Controller('claims')
+export class ClaimsController {
+  constructor(
+    private readonly claimsService: ClaimsService,
+    private readonly logger: PolymeshLogger
+  ) {
+    logger.setContext(ClaimsController.name);
+  }
+
+  @ApiOperation({
+    summary: 'Add Claims targeting an Identity',
+    description: 'This endpoint will add Claims to an Identity',
+  })
+  @ApiTransactionResponse({
+    description: 'Transaction response',
+    type: TransactionQueueModel,
+  })
+  @ApiTransactionFailedResponse({
+    [HttpStatus.UNPROCESSABLE_ENTITY]: ['Account does not have the required roles or permissions'],
```

Comment:
'A target Identity cannot have CDD claims with different IDs' this should also be added

Reply:
Also need to add `Some of the supplied Identity IDs do not exist` for NOT_FOUND